css/index.css

```
@@ -773,4 +772,215 @@ body {
 
 .food-spin__location-indicator{
   stroke: var(--title-color);
-}
\ No newline at end of file
+}
+
+.cart-summary {
+  max-width: 500px;
+  background-color: var(--color-white);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  top: 0%;
+  right: 0;
+  z-index: 1000;
+  height: 100%;
+  border: none;
+  padding: 0px;
+  --text-color: #FF922C;
+  --button-color: #FF6600;
+  --box-shopping: rgba(255, 102, 0, 0.25);
+  box-shadow: 0px 0px 70px 20px var(--box-shopping);
+  transform: translateX(200%);
+  transition: transform 0.4s ease;
+  opacity: 0;
+}
+
+.cart-summary[open] {
+  transform: translateX(0);
+  opacity: 1;
+}
+
+.cart-summary__close {
+  color: var(--text-color);
+  font-size: 24px;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.cart-summary__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  border-bottom: 3px solid #F5F5F5;
```

Si ya tienes el color definido, usalo
```suggestion
  border-bottom: 3px solid var(--color-whiteX);
```